pyproject.toml

```
@@ -53,7 +53,7 @@ dependencies = [
     "pyHanko>=0.12.1",
     "pyhanko-certvalidator>=0.19.5",
     "pypdf>=3.1.0",
-    "python-bidi>=0.4.2",
+    "python-bidi>=0.5.0",
```

```suggestion
    "python-bidi>=0.5.0,<0.6",
```
If this library introduces breaking changes on their minor versions, maybe we should update to 0.6.x only after making sure everything works?

---

They didn't introduce a breaking change a new minor version.
The breaking change appeared in python-bidi version 0.5.0
In version 0.4.2 the import works this way:
`from bidi.algorithm import get_display`
In version 0.5.0 the import works this way:
`from bidi import get_display`

The bug appeared because xhtml2pdf does not guard against major version changes of its dependencies.

---

@jeromecc if the second number is now the major, what is the first number?